drivers/serial/uart_cmsdk_apb.c

```diff
@@ -258,7 +259,18 @@ static int uart_cmsdk_apb_fifo_read(struct device *dev,
  */
 static void uart_cmsdk_apb_irq_tx_enable(struct device *dev)
 {
+	unsigned int key;
```

Comment:
u32_t ?

Reply:
If you grep thru Zephyr source code, you'll see that it's kind of Zephyr idiom to have that thing be typed as `unsigned int`, and named as `key`. Both don't make much sense, but everyone just copy-pastes them around, and I do the same.

Reply:
I do not like it as well but @pfalcon is right, it is quite conventional in Zephyr.